docs/core/install/dependencies.md

```diff
@@ -290,6 +290,24 @@ For .NET Core apps that use the *System.Drawing.Common* assembly, you'll also ne
 > of libgdiplus by adding the Mono repository to your system. For more information,
 > see <https://www.mono-project.com/download/stable/>.
 
+### Alpine
+
+Alpine distributions require the following libraries to be installed:
+
+- icu-libs (this is not needed if globalization is disabled)
+- krb5-libs
+- libcurl
+- libintl
+- libssl1.1 (for Alpine >= 3.9) or libssl 1.0 (for older ones)
```

Comment:
should it be without space like the 1.1 version?

```suggestion
- libssl1.1 (for Alpine >= 3.9) or libssl1.0 (for older ones)
```

Reply:
I edited this suggestion to add `or later` instead of `>=` for the version compare.

Reply:
Thank you, yes, there should be no space there.